docs/segment-store-service.md

```diff
@@ -30,14 +30,15 @@ Throughout the rest of this document, we will use the following terminology:
 
 # Architecture
```

Comment:
You had some nice illustrations that we could add here. Please consider adding and discussing them.

Reply:
The illustrations are still there.